packages/ui5-info/src/ui5-version-info.ts

```diff
@@ -357,7 +357,14 @@ export async function getUI5Versions(filterOptions?: UI5VersionFilterOptions): P
         const ui5Version: UI5Version = {
             version: ui5
         };
-        if (filterOptions?.includeDefault && defaultUI5Version === ui5) {
+        if (
+            filterOptions?.includeDefault &&
+            defaultUI5Version === ui5 &&
+            !filterOptions.markLatestMaintainedAsDefault
+        ) {
+            ui5Version.default = true;
+        }
+        if (filterOptions?.markLatestMaintainedAsDefault && latestUI5Version === ui5) {
```

Comment:
`default` should only ever be included where `includeDefault` is specified.

Reply:
Addressed in [8346988](https://github.com/SAP/open-ux-tools/pull/2166/commits/83469882db35845d335cfb76671fedaa209b415a), thank you @IainSAP.